finetuner/models.py

```diff
@@ -0,0 +1,158 @@
+import abc
+from typing import Any, Dict, List, Optional, Tuple, TypeVar
+
+ModelStubType = TypeVar('ModelStubType', bound='_ModelStub')
+
+
+class _ModelStub(metaclass=abc.ABCMeta):
```

Comment:
just curious, this base stub class has been excluded from the inspect, right?

Reply:
yes https://github.com/jina-ai/finetuner/pull/428/files#diff-8de89b78deef000f7d454bc8ec2e228d003253f07319dbd3e455eeb18da24a77R53